src/components/ContentViewer.jsx

```diff
@@ -22,11 +22,13 @@ function ContentViewer({
         {date + " "}
         <HeartCounter count={heartCount} />
       </div>
-      {!isDescription && <ImgWrapper>
-        <img src={url} />
-      </ImgWrapper>
-      {hasText && <TextContainer>{text}</TextContainer>}
-      </ImgWrapper>}
+      {!isDescription && <div>
+        <ImgWrapper>
+          <img src={url} />
+        </ImgWrapper>
+        {hasText && <TextContainer>{text}</TextContainer>}
+      </div>
+      }
```

Comment:
isDescription은 imageWrapper만 감싸게 됩니다!
```suggestion
      {!isDescription && <ImgWrapper>
          <img src={url} />
        </ImgWrapper>}
      {hasText && <TextContainer>{text}</TextContainer>}
```

Reply:
다시 리뷰하며 이상한 인덴팅과 기분이 들던 참이었습니다! 확인 감사합니다!!